### PR TITLE
Create order item model #27

### LINKED ISSUE #27: Create Order Item Model

### PR DESCRIPTION
**Pull request by JeffKandel**
closes #27

### CODE REVIEW

File: db/models/index.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ const app = require('APP')
       User: require('./user'),
       Thing: require('./thing'),
       Favorite: require('./favorite'),
+      Item: require('./item')
```
Comment (JeffKandel) [inline db/models/index.js]: I should move this down a line...

File: db/models/item.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const Sequelize = require('sequelize')
+
+// Including some additional flexibility in case a user logged in with oAuth
+module.exports = db => db.define('items', {
+  price: {
+    // e.g. 90210.00
+    type: Sequelize.DECIMAL(7, 2),
```
Comment (JeffKandel) [inline db/models/item.js:9]: DECIMAL will cause npm test headaches, but it's okay.  This is the preferred way to work with money on the web.  For the preferred way to work with money on the dark web see: https://bitcoin.org/en/

File: db/models/item.test.js [hunk 3]
```diff
@@ -17,7 +17,7 @@ describe('The `Item` model', () => {
   let item
   beforeEach(() => {
     item = Item.build({
-      price: 1000.00,
+      price: 1000.01,
```
Comment (JeffKandel) [inline db/models/item.test.js:20]: I have made some changes because of JS coercion issues that were coming up.  What do you think about it?

**Review by lada-tech — APPROVED** [review]
Add associations and it's good to go

**Review by JeffKandel — COMMENTED** [review]
All these are solid changes, but Lada should approve them.

**Pull request merged**